src/Corax/IndexWriter/EntriesModifications.cs

```diff
@@ -0,0 +1,354 @@
+using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using System.Text;
+using Corax.Utils;
+using Sparrow.Server;
+using Voron.Util;
+
+namespace Corax.IndexWriter;
```

Comment:
The namespace and the class name being the same is going to be pretty confusing.
Let's call it `Corax.IndexWriting` and then have `Corax.IndexWriting.IndexWriter` ?

Reply:
Maybe too late, but if we are going to be doing cleanup maybe we could do:  `Corax.Indexing` and `Corax.Searching` or `Corax.Querying` and unify the IndexSearcher and the Queries under that.

Reply:
I like the idea of `Corax.Indexing` and `Corax.Querying` namespaces. @maciejaszyk what do you think?

Reply:
Done :)